docs/migration/migratingfrommlapi.md

```diff
@@ -0,0 +1,111 @@
+---
+id: migratingfrommlapi
+title: Updating to the Unity Package
+sidebar_label: Updating to the Unity Package
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::important
+This upgrade guide targets project using the MLAPI version which was installed by the MLAPI installer or by cloning the source code from Github.
+If your project uses a Package Manager version of MLAPI then this guide is not relevant.
+:::
+
+The aqcuisiton of MLAPI has been a unique event in Unity history. In our efforts to integrate MLAPI into the Unity eco system we are turning it in a Unity package.
```

Comment:
```suggestion
The aqcuisiton of MLAPI has been a unique event in Unity history. In our efforts to integrate MLAPI into the Unity ecosystem, we are providing it in a Unity package.
```

Reply:
Oh a thought, we may want to add a prerequisite that they need to be on Unity 2020.3?

```
2020.3 and later (recommended) is required for Unity MLAPI. See [Unity Upgrade Guides](https://docs.unity3d.com/Manual/UpgradeGuides.html) if you need to upgrade prior to installation.
```